chair/role.html

```diff
@@ -73,6 +74,8 @@ <h3>Maintains Group Process &amp; Organization</h3>
 or her chair "hat" at that time.</li>
 <li>Keeps Group's charter compliant with W3C Process, and initiates
 charter update as needed.</li>
+<li>Keeps participants to abide with the terms and spirit of the
```

Comment:
Slightly odd wording for "Keeps participants to", I think 'keeps' should be replaced. I'm not sure whether it would be 'Encourages', 'Asks', or 'Requires', depending on the strength of the assertion?

Reply:
I used "Ensures"